lambda/test/source/GooglePlotSourceTest.js

```diff
@@ -0,0 +1,111 @@
+var expect = require('chai').expect;
+var googleSource = require('../../source/GoogleCustomSearchSource');
+
+describe('googleSourceTest', function() {
+    
+    it('should find The Mask', function() {
+        this.timeout(5000);
+        return googleSource.getMovies("guy wears a green mask").then((movies) => {
+            expect(movies.find((m) => {return m.getTitle().indexOf("The Mask") > -1 })).to.not.equal(undefined)
+        })
+    });
+
+    it('should find Pacific Rim', function() {
+        this.timeout(5000);
+        return googleSource.getMovies("giant robot fights giant alien").then((movies) => {
+            expect(movies.find((m) => {return m.getTitle().indexOf("Pacific Rim") > -1 })).to.not.equal(undefined)
+        })
+    });
+
+    // it('should find The Force Awakens', function() {
```

Comment:
lol might as well just delete these commented out test if it doesn't work for google sources